### ISSUE TITLE
Cufflinks ValueError

### ISSUE THREAD

**Issue by aayushmalik** [santosjorge/cufflinks#123]
As given on the Plotly Cufflinks tutorial, I am using the following code to generate grouped bar charts using a dataframe with the help of cufflinks

`cross_table.iplot(kind='bar')`

but I am getting this error always. How do I solve this?

```
ValueError: 
    Invalid value of type 'builtins.str' received for the 'color' property of layout.legend.font
        Received value: 'pearl06'
```

I have already written following code. 

```
import plotly as py
from plotly.offline import download_plotlyjs, init_notebook_mode, plot, iplot
init_notebook_mode(connected=True)
import plotly.graph_objs as go
import plotly.figure_factory as ff
import cufflinks as cf
cf.go_offline()
cf.set_config_file(offline=True, world_readable=False, theme='white')
```

I tried changing config_file with different themes, I am still getting the same error. 

My Python version is 3.6, Plotly version is 3.1.0, and cufflinks is 0.13.0
I am running the code in Jupyter Notebook, version 5.6

Detailed Error

```
ValueError: 
    Invalid value of type 'builtins.str' received for the 'color' property of layout.legend.font
        Received value: 'pearl06'

    The 'color' property is a color and may be specified as:
      - A hex string (e.g. '#ff0000')
      - An rgb/rgba string (e.g. 'rgb(255,0,0)')
      - An hsl/hsla string (e.g. 'hsl(0,100%,50%)')
      - An hsv/hsva string (e.g. 'hsv(0,100%,100%)')
      - A named CSS color:
            aliceblue, antiquewhite, aqua, aquamarine, azure,
            beige, bisque, black, blanchedalmond, blue,
            blueviolet, brown, burlywood, cadetblue,
            chartreuse, chocolate, coral, cornflowerblue,
            cornsilk, crimson, cyan, darkblue, darkcyan,
            darkgoldenrod, darkgray, darkgrey, darkgreen,
            darkkhaki, darkmagenta, darkolivegreen, darkorange,
            darkorchid, darkred, darksalmon, darkseagreen,
            darkslateblue, darkslategray, darkslategrey,
            darkturquoise, darkviolet, deeppink, deepskyblue,
            dimgray, dimgrey, dodgerblue, firebrick,
            floralwhite, forestgreen, fuchsia, gainsboro,
            ghostwhite, gold, goldenrod, gray, grey, green,
            greenyellow, honeydew, hotpink, indianred, indigo,
            ivory, khaki, lavender, lavenderblush, lawngreen,
            lemonchiffon, lightblue, lightcoral, lightcyan,
            lightgoldenrodyellow, lightgray, lightgrey,
            lightgreen, lightpink, lightsalmon, lightseagreen,
            lightskyblue, lightslategray, lightslategrey,
            lightsteelblue, lightyellow, lime, limegreen,
            linen, magenta, maroon, mediumaquamarine,
            mediumblue, mediumorchid, mediumpurple,
            mediumseagreen, mediumslateblue, mediumspringgreen,
            mediumturquoise, mediumvioletred, midnightblue,
            mintcream, mistyrose, moccasin, navajowhite, navy,
            oldlace, olive, olivedrab, orange, orangered,
            orchid, palegoldenrod, palegreen, paleturquoise,
            palevioletred, papayawhip, peachpuff, peru, pink,
            plum, powderblue, purple, red, rosybrown,
            royalblue, saddlebrown, salmon, sandybrown,
            seagreen, seashell, sienna, silver, skyblue,
            slateblue, slategray, slategrey, snow, springgreen,
            steelblue, tan, teal, thistle, tomato,  turquoise,
            violet, wheat, white, whitesmoke, yellow,
            yellowgreen
```
https://github.com/plotly/plotly.py/pull/956 This is the closest to what I found on GitHub.

**Comment by timkpaine** [comment]
duplicate of https://github.com/santosjorge/cufflinks/issues/119

**Comment by aayushmalik** [comment]
Downgraded to version 2.7.0, it is working now.

**Comment by santosjorge** [comment]
Fixed in v0.14
